src/components/Helpers.tsx

```diff
@@ -0,0 +1,24 @@
+import { FilterBy } from '../types/FilterBy';
+import { Todo } from '../types/Todo';
+
+export const filterTodos
+= (initialTodos: Todo[], filter:string, query: string) => {
+  const normalizedQuery = query.trim().toLowerCase();
+
+  const filteredTodos = initialTodos.filter((todo: Todo) => {
+    const isIncludes = todo.title.toLowerCase().includes(normalizedQuery);
```

Comment:
gramatically incorrect. For verbs we would do it like `doesInclude` or just `includes` is enough
But it is not clear this way, so I would come up with more descriptive naming